src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -3283,7 +3513,7 @@ impl Coordinator {
                 let plan = match mz_sql::plan::plan(
```

Comment:
Just about this we mark the ID of the `cur_entry` as unresolvable for planning, remind me again why we do that?

Reply:
Clarified this in a call. The issue here is that we need to re-plan the statement and want to ensure we don't get an "object with that name already exists" error.